DQM/L1TMonitor/src/L1TStage2uGMT.cc

```diff
@@ -54,9 +54,8 @@ void L1TStage2uGMT::bookHistograms(DQMStore::IBooker& ibooker, const edm::Run&,
     
     ugmtBMTFhwPhi = ibooker.book1D("ugmtBMTFhwPhi", "uGMT BMTF Input #phi", 71, -10.5, 60.5);
     ugmtBMTFhwPhi->setAxisTitle("Hardware #phi", 1);
-    ugmtBMTFhwPhi->getTH1F()->SetMinimum(0.0);
 
-    ugmtBMTFglbPhi = ibooker.book1D("ugmtBMTFglbhwPhi", "uGMT BMTF Input #phi", 576, -0.5, 575.5);
+    ugmtBMTFglbPhi = ibooker.book1D("ugmtBMTFglbhwPhi", "uGMT BMTF Input #phi", 596, -10.5, 585.5);
```

Comment:
Extending by just one bin should be enough. What is the reason to extend by 10 bins in both directions?